packages/wp-comments/src/__tests__/commentsHandler.test.ts

```diff
@@ -12,7 +12,10 @@ let store: InitializedStore<Packages>;
 let api: jest.Mocked<Api>;
 
 beforeEach(() => {
-  const packages: any = mergeDeepRight(clone(wpSource()), clone(wpComments));
+  const packages: any = mergeDeepRight(
+    mergeDeepRight(clone(wpSource()), clone(wpComments)),
+    { state: { router: { redirections: "no" } } }
+  );
```

Comment:
Why does `wp-comments` require setting `redirections` to "no"? Is not that the default value?

Reply:
The tests in wp-comments are calling `actions.source.fetch()` which is accessing `state.router.redirections`. This is the same case as mentioned in https://github.com/frontity/frontity/pull/601#discussion_r526495008.

It can be removed now that we're moving the redirections to `state.source.redirections`.

Reply:
Ok, thanks for the explanation 🙂